scripts/upgrade

```diff
@@ -158,17 +153,18 @@ then
     ynh_replace_string --match_string="__DBUSER__" --replace_string=$db_name --target_file="$rc_conf"
     db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
     ynh_replace_string --match_string="__DBPASS__" --replace_string="$db_pwd" --target_file="$rc_conf"
-    ynh_replace_string --match_string="__DBNAME__" --replace_string=$db_name --target_file="$rc_conf"
+    ynh_replace_string --match_string="__DBNAME__" --replace_string="$db_name" --target_file="$rc_conf"
+    ynh_replace_string --match_string="__LANGUAGE__" --replace_string="$language" --target_file="$rc_conf"
```

Comment:
You need to manage upgrade from previous version when `ynh_app_setting_get --app=$app --key=language` don't already exist

Reply:
@yalh76 Any hits on best way to do that? 🤔

Reply:
you can do like it's made for `$db_name`
https://github.com/YunoHost-Apps/roundcube_ynh/blob/f6183f057154ce38c5004f7056a7189536d71a04/scripts/upgrade#L37-L41